VMEncryption/main/SupportedOS.json

```diff
@@ -1,5 +1,11 @@
 {
   "redhat": [
+        {
+          "Version" : "8.1"
+        },
+        {
```

Comment:
8.0 for testing but remove since abandoned - comment this is only for test needs to be removed before deployment unless validated